Makefile-docker

```diff
@@ -82,45 +66,6 @@ update_assets:
 	# Collect static files: This MUST be run last or files will be missing
 	$(PYTHON_COMMAND) manage.py collectstatic --noinput
 
-.PHONY: update
-update: update_db update_assets ## update the dependencies, the database, and assets
-
-.PHONY: reindex
-reindex: ## reindex everything in elasticsearch, for AMO
-	$(PYTHON_COMMAND) manage.py reindex $(ARGS)
-
-.PHONY: setup-ui-tests
-setup-ui-tests:
```

Comment:
Ah TIL, looked like dead code. Instead I guess we should add a link to addons-frontend to make it clear.

Reply:
Looks like we need to figure out what this target was doing that is not now in make up internally. It might be that we can still remove it or make it a noop until it's removed from FE testing.

Reply:
Yeah. It used to require creating users and so on, but that no longer applies. The only thing they really need is a working local environment with a homepage that loads, and that's it.

Reply:
I think ultimately rather than spending too much more time to use the correct `make up` command we should fix that test to no longer depend on addons-server (https://github.com/mozilla/addons/issues/2108)

Reply:
Fair point as well. So I will make this an empty command so it doesn't straight up break the frontend and we can independently fix the frontend to do the right thing. fwiw the correct make up is just `make up INIT_CLEAN=true` so wouldn't be much trouble.

Reply:
Done